yarn.lock

```diff
@@ -4664,19 +4655,6 @@ flatted@^3.1.0:
   resolved "https://registry.yarnpkg.com/flatted/-/flatted-3.2.7.tgz#609f39207cb614b89d0765b477cb2d437fbf9787"
   integrity sha512-5nqDSxl8nn5BSNxyR3n4I6eDmbolI6WT+QqR547RwxQapgjQBmtktdP+HTBb/a/zLsbzERTONyUB5pefh5TtjQ==
 
-"flowgen@git+https://git@github.com/Khan/flowgen.git#fa8050929948fb4313f738301285fb44625678c4":
```

Comment:
It looks like this wasn't cleaned up properly when this was removed? cc: @kevinbarabash

Reply:
I suspect it was removed from the `package.json` by hand but `yarn` wasn't re-run to update `yarn.lock`.